comms/src/message/inbound_message.rs

```diff
@@ -20,34 +20,25 @@
 //  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 //  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-use crate::{
-    message::{Frame, MessageEnvelopeHeader},
-    peer_manager::Peer,
-};
-use serde::{Deserialize, Serialize};
+use crate::{message::MessageEnvelopeHeader, peer_manager::Peer};
 
-/// The InboundMessage is the container that will be dispatched to the domain handlers. It contains the received
-/// message and source identity after the comms level envelope has been removed.
-#[derive(Clone, Debug, Serialize, Deserialize)]
+/// Authenticated inbound message
+#[derive(Clone, Debug)]
 pub struct InboundMessage {
-    /// The message envelope header
+    /// The deserialized message envelope header
     pub envelope_header: MessageEnvelopeHeader,
     /// The connected peer which sent this message
     pub source_peer: Peer,
-    /// The version of the incoming message envelope
-    pub version: u8,
     /// The raw message envelope
-    pub body: Frame,
+    pub body: Vec<u8>,
```

Comment:
In MessageEnvelope Frames and Framesets are still used, shouldn't this also be a Frame.

Reply:
Frame/FrameSet were indented to represent Frames coming from zmq connections. Once a frame has been deserialized, any "byte blob" field is not a Frame.